### DISCUSSION TITLE
Captain Rumsey's Lager

### DISCUSSION THREAD

**Discussion by TheDaVaiLama** [chromiecraft/chromiecraft#6485] [Suggestions]
Hi how are ya. Much like many other cooking/fishing achievement hunters, months later after doing the fishing daily every day, I am still missing the Captain Rumsey's Lager recipe. Now taking into consideration 0.14% drop rate is pretty darn low, I was hoping someone could consider adding the recipe to the list of items one can get with mythical points. It might even be a semi-unreasonable amount of mythic points, to reflect the low drop rate, however I really do feel it could really help out those of us dreading that they would be doing this quest way past the 74 bracket

**Comment by Hihilist** [comment]
See https://github.com/chromiecraft/chromiecraft/issues/5518 and https://github.com/chromiecraft/chromiecraft/issues/6325 - it should drop but bugs were never looked into.

**Reply by TheDaVaiLama** [reply]
Hey, thanks for answering! This is troublesome, however I do feel like reinforces my point/suggestion even more? I hope someone (who is a factor in the decision making process) takes this suggestion into consideration!
